src/Controllers/FoodVoucherController.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Engelsystem\Controllers;
+
+use Engelsystem\Config\Config;
+use Engelsystem\Helpers\Authenticator;
+use Engelsystem\Helpers\Carbon;
+use Engelsystem\Helpers\Translation\Translator;
+use Engelsystem\Helpers\UserVouchers;
+use Engelsystem\Http\Exceptions\HttpForbidden;
+use Engelsystem\Http\Exceptions\HttpNotFound;
+use Engelsystem\Http\Redirector;
+use Engelsystem\Http\Request;
+use Engelsystem\Http\Response;
+use Engelsystem\Models\User\User;
+use Engelsystem\Models\Worklog;
+use ErrorException;
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\Exception\ConnectException;
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Log\LoggerInterface;
+
+class FoodVoucherController extends BaseController
+{
+    use HasUserNotifications;
+
+    public function __construct(
+        protected Authenticator $auth,
+        protected GuzzleClient $guzzle,
+        protected Config $config,
+        protected LoggerInterface $log,
+        protected Worklog $worklog,
+        protected Redirector $redirect,
+        protected Response $response,
+        protected User $user,
+        protected Translator $translator,
+    ) {
+    }
+
+    private function checkActive(): void
+    {
+        if (!$this->config->get('enable_voucher') || !$this->config->get('enable_force_food')) {
```

Comment:
```suggestion
        $this->checkConfig();
```
ans integrate it there?

Reply:
The config should either be complete and active or the whole functionality of all methods would not be needed and thus can 404.